pandas/core/frame.py

```diff
@@ -5891,17 +5891,22 @@ def _construct_result(self, result) -> "DataFrame":
     The resulting index will be a MultiIndex with 'self' and 'other'
     stacked alternately at the inner level.
 
+Raises
+------
+ValueError
+    * When `errors='raise'` and both DataFrames don't have identical labels.
```

Comment:
compare doesn't have a errors parameter. doesn't this raise ValueError always if both DataFrames don't have identical labels?

Reply:
True, made an edit, hopefully I understood you correctly.